_content/av/entrance-into-light-of-the-dharma_anandajoti.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Dharmāloka-Mukhaṁ: The Entrance into the Light of the Dharma"
+translator: anandajoti
+editor: "S. Lefman"
+external_url: "https://ancient-buddhist-texts.net/Texts-and-Translations/Short-Pieces-in-Sanskrit/Dharmaloka-Mukham.pdf"
```

Comment:
The htm please, not the pdf:  https://ancient-buddhist-texts.net/Texts-and-Translations/Short-Pieces-in-Sanskrit/Dharmaloka-Mukham.htm